Makefile

```diff
@@ -1,48 +1,67 @@
-JSONNET = https://github.com/google/jsonnet/releases/download/v0.17.0/jsonnet-bin-v0.17.0-linux.tar.gz
+JB = https://github.com/jsonnet-bundler/jsonnet-bundler/releases/latest/download/jb-linux-amd64
 BINDIR = bin
 TEMPLATESDIR = templates
 OUTPUTDIR = rendered
 ALLDIRS = $(BINDIR) $(OUTPUTDIR)
 SYNCER_IMG_TAG ?= quay.io/cloud-bulldozer/dittybopper-syncer:latest
 PLATFORM = linux/amd64,linux/arm64,linux/ppc64le,linux/s390x
 
-# Get all templates at $(TEMPLATESDIR)
-TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*.jsonnet)
+ifeq ($(filter v2,$(MAKECMDGOALS)),v2)
+  	# Set variables and instructions for v2
+  	TEMPLATES := $(wildcard $(TEMPLATESDIR)/**/*-v2.jsonnet)
+	LIBRARY_PATH := $(TEMPLATESDIR)/vendor
+	JSONNET := https://github.com/cloud-bulldozer/utils/releases/download/v0.0.0/jsonnet-bin-v0.20.0-linux.tar.gz
```

Comment:
https://github.com/cloud-bulldozer/utils/releases/download/v0.0.0/jsonnet-bin-v0.20.0-linux.tar.gz

Curious why we are keeping this in the utils directory versus pulling for an official source?

Reply:
That is to avoid breaking changes due to upstream updates which usually happens in grafonnet's case. When ever we need an updated one, we can simply update it here, which should be very less frequent.

Reply:
?, what does grafonnet has to do with the jsonnet version?, let's keep the old version URL. using this approach would require to update the version twice

Reply:
Jsonnet binary is used to compile grafonnet code. So the older version of jsonnet binary was not compatible with new grafonnet library and viceversa.

In order to avoid those breaking changes I have built latest jsonnet binary using their source code because official jsonnet repo has stopped publishing binaries in their releases: https://github.com/google/jsonnet/releases.

Also I have fixed its version so that we don't run into breaking changes while building jsons for our dashboards. We can update it in future if required, but it should be very frequent may be once in a 2 years or so which I thought should be fine instead of other operational overhead.

Reply:
Could we use something like https://github.com/google/go-jsonnet/releases/ 

Looking at some issues I saw : https://github.com/google/jsonnet/issues/958

Reply:
Thanks for sharing. I wonder one how did I even miss this one :)